contents/tutorials/framer-analytics.md

```diff
@@ -0,0 +1,112 @@
+
+Once on a paid plan, go to the "General" tab in site settings and scroll down to the "Custom Code" section. Under "End of `<head>` tag", paste your PostHog JavaScript snippet here. Make sure to press "Save" next to the "Custom Code" heading. 
+
+![Script](../images/tutorials/framer-analytics/script.png)
+
+Once you publish and go to your site, PostHog starts to autocapture events like pageviews, button clicks, and inputs for your site.
+
+![Autocapture](../images/tutorials/framer-analytics/autocapture.png)
+
+### Enabling session replays
+
+To enable session replays, go to [your project settings](https://app.posthog.com/project/settings#recordings), scroll down to "Recordings," and enable "Record user sessions." PostHog then starts to record sessions of your users, so you can see all the details of how they use your site.
+
+## Adding a custom event
+
+You can add [custom code components](https://www.framer.com/developers/#code-components) to Framer and because of that, we can set up [custom event capture](/docs/libraries/js#capturing-events). We will create a basic button that captures a `clicked homepage button`  and sets a `clicked_homepage_button` person property to `true`.
+
+To do this, go to the "Assets" tab in the top left of your project. Next to the "Code" tab, click the plus icon to create a code file. Name the file `CaptureButton`, set it as a "New component" and press "Create."
+
+Once done, delete the boilerplate code and add a button that captures an event with `window.posthog()` like this:
+
+```js
+export default function CaptureButton() {
+    const handleClick = () => {
+        window.posthog.capture("clicked homepage button", {
+            $set_once: { clicked_homepage_button: true },
+        })
+    }
+
+    return <button onClick={handleClick}>Click me</button>
+}
+```
+
+After pasting this in, press cmd/ctrl + S to save the component, and go back to the home page by clicking "Home" above the code. Once on the home page, drag and drop the `CaptureButton` component from the sidebar into your page and publish again to update your site with your new button.
+
+![Framer button video](../images/tutorials/framer-analytics/framer-button.mp4)
+
+Once updated, you can head to your site and click the button to see events captured in PostHog and person properties set on the user.
+
+![Custom event and person properties](../images/tutorials/framer-analytics/person.png)
+
+## Customizing components with feature flags
+
+Similarly, we can use [feature flags](/docs/feature-flags) in our custom code components. To do this, first, go to the [feature flag tab](https://app.posthog.com/feature_flags) in PostHog. Click "New feature flag," enter a key (I choose `framer-example`), fill out the details, set release conditions to roll out to 100% of users, and click "Save."
```

Comment:
Perhaps we should give one line sentence of what feature flags do? or why you'd want to add them